modules/ding_frontend/ding_frontend.module

```diff
@@ -268,3 +268,61 @@ function ding_frontend_search_api_views_query_alter(view &$view, SearchApiViewsQ
     }
   }
 }
+
+function ding_frontend_form_ting_search_sort_form_alter(&$form, &$form_state, $form_id) {
+  $form['#attributes']['class'][] = 'form-search-sort';
+  $form['#attributes']['class'][] = 'js-form-search-sort';
+
+  $form['sort']['#prefix'] =
+    '<label class="form-search-sort__prefix">'
+      . t('Sorted by') .
+    '</label>';
```

Comment:
Could this be handled by setting an appropriate `#title` for the `sort` element? Note that this would not have the appropriate classes.